README.md

```diff
@@ -1,4 +1,48 @@
-yolo-hipster
-============
+taxster
+=======
 
-Naive Bayes taxonomy classifier using scikit-learn
+This library contains tools for working with taxonomic assignment data. This should be considered pre-alpha software. **Public APIs can and will change!**
+
+Install
+-------
+
+pip install https://github.com/biocore/taxster/archive/master.zip
+
+Consensus taxonomy assignments from .uc files
+---------------------------------------------
+
+This example illustrates how to apply the consensus taxonomy processing described in [Bokulich et al (in preparation)](https://peerj.com/preprints/934/). It assumes that you have the test data files included in this repository in ``./test-data``.
+
+```python
+import taxster
+
+taxonomy_map = {}
+for line in open('./test-data/uc/tax-map.tsv', 'U'):
```

Comment:
Pandas will read the entire file which can require a lot of memory. I only use it when I need to do actual data-frame stuff. You could use the csv parser in the standard library.

--austin

> On Mar 2, 2016, at 08:57, Daniel McDonald <notifications@github.com> wrote:
> 
> In README.md:
> 
> > +
> > +Install
> > +-------
> > +
> > +pip install https://github.com/biocore/taxster/archive/master.zip
> > +
> > +Consensus taxonomy assignments from .uc files
> > +---------------------------------------------
> > +
> > +This example illustrates how to apply the consensus taxonomy processing described in [Bokulich et al (in preparation)](https://peerj.com/preprints/934/). It assumes that you have the test data files included in this repository in ``./test-data``.
> > +
> > +```python
> > +import taxster
> > +
> > +taxonomy_map = {}
> > +for line in open('./test-data/uc/tax-map.tsv', 'U'):
> totally minor, but would it be more concise to use pandas?
> 
> —
> Reply to this email directly or view it on GitHub.
>